README.md

```diff
@@ -196,6 +196,22 @@ if err != nil {
 }
 ```
 
+#### Update MaxMessageSize/read limit for realtime connection
+- Connection MaxMessageSize limit is automatically configured based on `ConnectionDetails.MaxMessageSize` property received when connection is successfully made with ably.
```

Comment:
I don't think we need to talk about `ConnectionDetails` here, it would be simpler to just say this:
```suggestion
- The default `MaxMessageSize` is automatically configured by Ably when connection is established with Ably.
```